src/Run - Static.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this icon necessary?

Reply:
No, that was my mistake. Removed this one and the "Loading"